.github/workflows/multiqc.yml

```diff
@@ -68,12 +66,16 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      # For CSP checking
-      - name: Install dependencies for CI tests
-        run: python -m pip install --upgrade ${{ env.core_python_dependencies }}
+      - uses: yezz123/setup-uv@v1
+
+      - name: Set the VIRTUAL_ENV variable for uv to work
+        run: echo "VIRTUAL_ENV=${Python_ROOT_DIR}" >> $GITHUB_ENV
 
       - name: Install MultiQC
-        run: pip install .
+        run: |
+          # Temporary hack to remove duplicated pip package (hopefully will be addressed in uv or GitHub)
```

Comment:
Would be good to link to a GitHub issue here in the code comment, so that it's easy to check on its status in the future easily.